src/main/java/Todo.java

```diff
@@ -0,0 +1,9 @@
+public class Todo extends Task{
+    public Todo(String description){
```

Comment:
I think it is better stylistically to put a space between the round and curly brackets